lib/index.js

```diff
@@ -10,6 +11,20 @@ exports.register = (server, options, next) => {
 	handler: require(`${controllerDir}/status`).status
   });
 
+  server.route({
+    method: 'POST',
+    path: '/login',
+    config: {
+      validate: {
+        payload: joi.object().keys({
+          username: joi.string().min(3).max(32).required(),
+          password: joi.string().min(8).max(64).required()
+        })
+      }
+    },
+    handler: require(`${controllerDir}/login`).login
```

Comment:
This is a great explanation, thanks!  I had not used template strings in node before and this is a great use case.